apps/src/applab/dataBrowser/dataStyles.js

```diff
@@ -15,12 +16,21 @@ export const row = {
 export const cell = {
   padding: cellPadding,
   border: '1px solid gray',
-  fontSize: 14
+  fontSize: 14,
+  maxWidth: 350,
+  overflow: 'hidden',
 };
 
-export const buttonCell = [cell, {
-  width: buttonCellWidth,
+export const addButtonCell = [cell, {
+  minWidth: addButtonCellWidth,
   backgroundColor: color.lightest_teal,
+  width: addButtonCellWidth,
+}];
+
+export const editButtonCell = [cell, {
+  minWidth: editButtonCellWidth,
+  backgroundColor: color.lightest_teal,
+  width: editButtonCellWidth,
```

Comment:
I've updated this to use `Object.assign`, but I don't believe the destructuring syntax will work in this case because we are not actually destructuring:
```
> var obj = {a:1, b:2}
< undefined
> {...obj, b:3}
< VM1413:1 Uncaught SyntaxError: Unexpected token ...
```
The `...` aka [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator) can also be used to create arrays, but apparently not for creating objects.